src/commonComponents/iconWithLabel/IconWithLabel.js

```diff
@@ -0,0 +1,34 @@
+// Library
+import PropTypes from 'prop-types'
+
+// css
+import './iconWithLabel.css'
+
+function FavouriteFilter(props){
+
+    const { icon, label, containerClassName, imgClassName } = props;
```

Comment:
pass a single className, and that should target your child elements too by using css descendant selector